File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/StudTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;
import java.util.Set;

public class StudTag extends Tag {
    public StudTag(int count) {
        super(count, Constants.STUD_TAG, Constants.STUD_TAG_REMOTE, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/StudTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class StudTag(count: Int) : Tag(count, Constants.STUD_TAG, Constants.STUD_TAG_REMOTE, HashSet(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))